File: BE/health-service/src/main/java/com/s206/health/nutrition/meal/dto/response/MealTimeNutritionResponse.java
package com.s206.health.nutrition.meal.dto.response;

import com.s206.health.nutrition.meal.entity.MealType;
import lombok.Builder;
import lombok.Getter;

import java.time.LocalDateTime;
import java.time.LocalTime;
import java.util.List;

@Getter
@Builder
public class MealTimeNutritionResponse {
    private Integer mealTimeId;
    private MealType mealType;
    private LocalTime eatingTime;
    private List<MealFoodResponse> foods;
    private MealNutritionDto nutrition;
    private String mealTimeImageUrl;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: FE/mobile/src/main/java/com/example/diaviseo/network/meal/dto/res/MealTimeNutritionResponse.kt
package com.example.diaviseo.network.meal.dto.res

// 아침, 점심, 저녁, 간식의 식단 응답형태
data class MealTimeNutritionResponse(
    val mealTimeId: Int?,
    val mealType: String,
    val eatingTime: String,
    val foods: List<MealFoodResponse>,
    val nutrition: MealNutritionResponse,
    val createdAt: String,
    val updatedAt: String,
    val mealTimeImageUrl: String?
)

